esp/services/ws_dfu/ws_dfuService.cpp

```diff
@@ -2298,6 +2298,10 @@ void CWsDfuEx::doGetFileDetails(IEspContext &context, IUserDescriptor *udesc, co
                 if (version >= 1.34)
                 {
                     __int64 diff = size - compressedSize;
+                    // This is slightly strange - compressionRatio = compressedSize/size would be more intuitive.
```

Comment:
perhaps would changing to more intuitive/consistent with elsewhere - compressedSize/size in this PR?